File: College Courses/CIS 159 Visual Basic/Modules/Module 10/Kayak Browser/prjKayakBrowser/prjKayakBrowser-Kolar/prjKayakBrowser-Kolar/Form1.vb
'Alex Kolar
'4/18/2013 Redone on 4/22/2013
'CIS159 Section 28998
'Kayak Browser

Public Class Form1

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        'TODO: This line of code loads data into the 'KayaksDataSet.Kayak_Types' table. You can move, or remove it, as needed.
        Me.Kayak_TypesTableAdapter.Fill(Me.KayaksDataSet.Kayak_Types)

    End Sub

    Private Sub lstTypes_SelectedIndexChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles lstTypes.SelectedIndexChanged
        lblDescription.Text = lstTypes.SelectedValue.ToString()
    End Sub
End Class

File: College Courses/CIS 162 C#/Homework/2nd Attempt/CS14-Kolar/CS14-Kolar/Form1.cs
/*Alex Kolar
 * 5/1/2013
 * CIS162AD Section 29153
 * CS14 - Databases
 */

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace CS14_Kolar
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void tblTransactionsBindingNavigatorSaveItem_Click(object sender, EventArgs e)
        {
            this.Validate();
            this.tblTransactionsBindingSource.EndEdit();
            this.tableAdapterManager.UpdateAll(this.dbTranscriptsDataSet);

        }

        private void Form1_Load(object sender, EventArgs e)
        {
            // TODO: This line of code loads data into the 'dbTranscriptsDataSet.tblTransactions' table. You can move, or remove it, as needed.
            this.tblTransactionsTableAdapter.Fill(this.dbTranscriptsDataSet.tblTransactions);

        }
    }
}
